packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -661,7 +663,9 @@ export class Virtualizer<T extends object, V, W> {
         }
 
         let item = this.getItem(visibleLayoutInfos.get(key).key);
-        if (view.content === item) {
+        // always need to rerender columns so that the resizer stuff gets rerendered correctly
```

Comment:
better to give a concrete example I think than "stuff"
```suggestion
        // always need to rerender columns so that the resizer aria-values get updated
```